.node-version

```diff
@@ -1 +1 @@
-20.10.0
+20.13.1
```

Comment:
lerna refused to build on `20.10.0`

https://github.com/pixiv/three-vrm/actions/runs/9123041637/job/25084897151?pr=1405#step:5:64